mappings/net/minecraft/entity/mob/PiglinBrain.mapping

```diff
@@ -119,7 +122,7 @@ CLASS net/minecraft/class_4838 net/minecraft/entity/mob/PiglinBrain
 		ARG 0 piglin
 	METHOD method_24781 stopWalking (Lnet/minecraft/class_4836;)V
 		ARG 0 piglin
-	METHOD method_24782 getAngryAt (Lnet/minecraft/class_4836;)Ljava/util/Optional;
+	METHOD method_24782 becomeAngryAt (Lnet/minecraft/class_4836;)Ljava/util/Optional;
```

Comment:
tbh...you don't really become angry *at* someone, you become angry _with_ them.

Reply:
Apparently @Prospector is right: https://english.stackexchange.com/a/43676

"angry at" is when you're angry about a situation, "angry with" is for people.

Reply:
I stumbled across that too when googling about it, though I'm really not sure. I feel like angry towards could also work well.

Reply:
Maybe `rageAt`

Reply:
Isn't this method just retrieving which entity the piglin is angry at? It's not performing any action in itself

Reply:
uh oh i think i misread something

Reply:
But it can stay the same